catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -366,7 +367,7 @@ private UpdateRequest rewriteRequestToAvoidHistoryConflicts(UpdateRequest update
                 updateRequest.getStoreIds());
     }
 
-    private boolean foundAllUpdateRequestMetacards(UpdateRequest updateRequest,
+    protected boolean foundAllUpdateRequestMetacards(UpdateRequest updateRequest,
```

Comment:
❗️ Since we're only changing the visibility so we can override this in the unit tests, we should make the method package-private instead and add a comment explaining why it isn't `private`.